app/src/main/java/com/severianfw/picto/data/remote/PhotosResponse.kt

```diff
@@ -0,0 +1,102 @@
+package com.severianfw.picto.data.remote
+
+import com.google.gson.annotations.SerializedName
+
+data class PhotosResponse(
```

Comment:
I think we should only consume fields that we're needed to reduce GSON Serialization overhead performance